### ISSUE TITLE
Windows installer creates desktop shortcut in current users desktop folder

### ISSUE THREAD

**Issue by websafe** [LibreCAD/LibreCAD#695]
The Windows installer creates the desktop shortcut in the current users desktop folder instead of `%PUBLIC%\Desktop`.

**Comment by lordofbikes** [comment]
I have to check, if there are any side effects with user settings.
Then I'd prefer a solution to let the user decide where to create the short cut, installing for the current or all users.

**Comment by websafe** [comment]
Yes, let the user decide but if it's possible please allow silent installers to choose `%PUBLIC%\Desktop` for desktop icons, because otherwise installing on multiple machines via `chocolatey` requires additional scripting.

In corporate environments users have no privileges to install software so admins have to move this shortcut every time to `%PUBLIC%\Desktop` and after each upgrade an additional shortcut on administrators desktop is created, which needs to be removed :-)

You should probably also check the discussion on https://chocolatey.org/packages/librecad :-)

**Comment by r-a-v-a-s** [comment]
http://forum.librecad.org/Windows-installer-sends-users-to-the-donation-page-td5712319.html
